tests/bluetooth/tester/src/btp.c

```diff
@@ -96,8 +96,8 @@ static void cmd_handler(void *p1, void *p2, void *p3)
 
 		cmd = k_fifo_get(&cmds_queue, K_FOREVER);
 
-		LOG_DBG("cmd service %u opcode %u index %u", cmd->hdr.service, cmd->hdr.opcode,
-			cmd->hdr.index);
+		LOG_DBG("cmd service 0x%02X opcode 0x%02X index 0x%02X", cmd->hdr.service,
```

Comment:
The BTP spec uses lower-case hex, so seems like `0x%02x` would be more appropriate. I believe it's the common form in most of the Bluetooth code base too.